pkg/utils/crd/crd.go

```diff
@@ -0,0 +1,226 @@
+package crd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/rancher/wrangler/v3/pkg/apply"
+	"github.com/rancher/wrangler/v3/pkg/crd"
+	"github.com/rancher/wrangler/v3/pkg/name"
+	"github.com/sirupsen/logrus"
+	apiext "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/rest"
+)
+
+// cloned from pkg/harvester/util/crd/init.go, to break loop importing
+
+// FromGV returns an abstract namespace-scope CRD handler via group version and kind.
+func FromGV(gv schema.GroupVersion, kind string, obj interface{}) crd.CRD {
+	return crd.FromGV(gv, kind).WithSchemaFromStruct(obj)
+}
+
+// NonNamespacedFromGV returns a cluster-scope CRD abstract handler via group version and kind.
+func NonNamespacedFromGV(gv schema.GroupVersion, kind string, obj interface{}) crd.CRD {
```

Comment:
this seems to be unused

Reply:
the crd/crd.go is copied from Harvester and kept unchanged & same with Harvester,  mainly for breaking the loop-importing between Harvester and network-controller

this function has no caller at the moment, but will be kept to be on sync with Harvester